.github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -5,17 +5,30 @@ title: '[PROPOSAL]'
 labels: proposal
 assignees: ''
 ---
-**What kind of business use case are you trying to solve? What are your requirements?**
-A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
 
-**What is the problem? What is preventing you from meeting the requirements?**
-Describe what impact this problem has for lack of a solution.
+## What are you trying to solve?
+_In one or two sentences, describe the feature and its core capabilities. e.g. My customers are asking for [...]_
 
-**What are you proposing? What do you suggest we do to solve the problem or improve the existing situation?**
-Describe your proposed solution. It's OK if you don't have one.
+## Why this?
+_Describe the most important user needs, pain points, and the value that this feature will bring to the OpenSearch community, as well as what impact it has if it isn't built. What is preventing you from meeting this need today?_
 
-**What are your assumptions or prerequisites?**
-Describe any assumptions you may be making that would limit the scope of this proposal.
+### How do you know?
+_Highlight any research, proposals, requests, issues, forum posts, anecdotes that signal this is the right thing to build. Highlight opportunities for additional research._
 
-**What are remaining open questions?**
-List questions that may need to be answered before proceeding with an implementation.
+## What matters most to OpenSearch?
+_Describe the value that this will bring to the OpenSearch project._
+
+## What does the customer experience look like?
```

Comment:
Don't use the word customer. 

I do like putting user experience here, so "What is the user experience?"